101-cosmosdb-create-arm-template/metadata.json

```diff
@@ -0,0 +1,7 @@
+{ 
+   "itemDisplayName": "Create an Azure Cosmos DB API account", 
+   "description": "This template creates an Azure Cosmos DB API account",
```

Comment:
Add something here about the different api types - e.g. just grab some of the text from the readme

Reply:
Done.